src/content/docs/distribute/app-store.mdx

```diff
@@ -157,6 +157,16 @@ Create a Info.plist file in the src-tauri folder:
 <dict>
 	<key>ITSAppUsesNonExemptEncryption</key>
 	<false/> # or `true` if your app uses encryption
+	<key>DTPlatformName</key>
```

Comment:
are these values just referencing the system that built the app bundle? or does `DTSDKName` reflect in the minimum supported version? i think it's just metadata right?

Reply:
also are you sure we do not need `DTPlatformBuild`? research online pointed we might need it